core/src/services/gcs/backend.rs

```diff
@@ -405,6 +407,50 @@ impl Accessor for GcsBackend {
         ))
     }
 
+    async fn copy(&self, from: &str, to: &str, _: OpCopy) -> Result<RpCopy> {
+        let source = percent_encode_path(&build_abs_path(&self.root, from.trim_end_matches("/")));
+        let dest = percent_encode_path(&build_abs_path(&self.root, to.trim_end_matches("/")));
+        let req_uri = format!(
+            "{}/storage/v1/b/{}/o/{}/rewriteTo/b/{}/o/{}",
```

Comment:
For now, let's utilize the `copyTo` function instead of `rewriteTo`. The latter is not suitable for retry operations and copying large files requires a different approach.

Reply:
Makes sense, will fix it later before night.